spec/integration/outputs/index_spec.rb

```diff
@@ -55,10 +56,18 @@
   subject { LogStash::Outputs::ElasticSearch.new(config) }
   
   let(:es_url) { "http://#{get_host_port}" }
-  let(:index_url) {"#{es_url}/#{index}"}
-  let(:http_client_options) { {} }
-  let(:http_client) do
-    Manticore::Client.new(http_client_options)
+  let(:index_url) { "#{es_url}/#{index}" }
+
+  let(:curl_opts) { nil }
+
+  def curl_and_get_json_response(url, method: :get); require 'open3'
+    stdout, status = Open3.capture2("curl #{curl_opts} -X #{method.to_s.upcase} -k #{url}")
```

Comment:
we already rely on `curl` being present in our integration test setup

Reply:
If `curl` is not installed `capture2` raise `Errno::ENOENT` . Could we catch that and have an assertion that cleanly exposes the problem of the missed tool. 
Being explicit would have trouble shooting a possible problem when tested on OSes that doesn't provide it.